packages/repack/commands.js

```diff
@@ -1,5 +1,6 @@
 const path = require('path');
 const { createRequire } = require('module');
+const { startCommandOptions } = require('./src/commands/utils/commandsOptions');
```

Comment:
when the package is built, this will still point to src and target TS file and fail after evaluation - it should point to `dist` instead, where JS will be stored. But then we will have the same problem in dev - I think we just need to inline the commandsOptions instead of having a separate file for that